src/questions/multipleChoiceQuestions.ts

```diff
@@ -11,7 +11,8 @@ export const multipleChoiceQuestions : Question[]= [
             '95'
         ], 
         correctAnswers : ['92'],
-        type: QuestionType.MULTIPLE_CHOICE
+        type: QuestionType.MULTIPLE_CHOICE,
+        repeatQuestion : "yes"
```

Comment:
Why would a question know whether it was already answered or not?

Also .. whenever you see boolean flags, a warning should rise in your head. This could potentially be code smell.

Reply:
https://understandlegacycode.com/blog/what-is-wrong-with-boolean-parameters/